### ISSUE TITLE
store.load not working

### ISSUE THREAD

**Issue by nicola** [antoniogarrote/rdfstore-js#98]
I really could not get this working with the current version of rdfstore, while it works with some very old one (0.8 I think)

This is my code:

```javascript
rdfstore.create(function (err, store) {
  store.load(mimeType, file, function(err, loaded) {
    console.log("LOADSTORE", err, loaded)
```

and I get `loaded = 0`

my `file` is:

```
@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .

<>
    <http://purl.org/dc/terms/title> "WebID profile of Nicola Greco" ;
    a <http://xmlns.com/foaf/0.1/PersonalProfileDocument> ;
    <http://xmlns.com/foaf/0.1/maker> <#me> ;
    <http://xmlns.com/foaf/0.1/primaryTopic> <#me> .

<#me>
    a <http://xmlns.com/foaf/0.1/Person> ;
    <http://www.w3.org/ns/auth/cert#key> <#key> ;
    <http://www.w3.org/ns/pim/space#preferencesFile> <../Preferences/prefs> ;
    <http://www.w3.org/ns/ui#backgroundImage> <van%20gogh%20gigapixel.png> ;
    <http://xmlns.com/foaf/0.1/img> <nicola%20machine%20drawing.jpg> ;
    <http://xmlns.com/foaf/0.1/knows> <http://melvincarvalho.com/#me> ;
    <http://xmlns.com/foaf/0.1/mbox> <mailto:me@nicolagreco.com> ;
    <http://xmlns.com/foaf/0.1/name> "Nicola Greco" .
```

Still, it loades it in the older version :( Any idea?

**Comment by ckristo** [comment]
Did you try to perform a query e.g. to see if the data really wasn't loaded? The `loaded` parameter is always zero for me too, but the triples are loaded anyway...

**Comment by nicola** [comment]
This is clearly a bug,
see my PR https://github.com/antoniogarrote/rdfstore-js/pull/99

**Comment by ckristo** [comment]
The PR was merged by @antoniogarrote -- the next release should fix this bug.

**Comment by antoniogarrote** [comment]
@ckristo @nicola 

I have just published version 0.9.7 with the fix.
I double checked your example and seem to be working correcly.

Cheers.
